example/ram/compile.lurk

```diff
@@ -0,0 +1,71 @@
+(defmacro or (a b)
+  `(if ,a t ,b))
```

Comment:
This makes me realize that a traditional short-circuiting' OR (one that immediately returns its first true branch) defined as a macro incurs a performance cost. It might be worth looking into the cost of a built-in (but optimized) version. NOTE: this is a tangent not directly related to the code you wrote, since you avoided the issue by returning T.